src/hooks/useActionCenterSubscribers/useActionCenterSubscribers.ts

```diff
@@ -12,5 +13,7 @@ export const useActionCenterSubscribers = () => {
   useAppUpdateActionSubscriber()
   useRfoxClaimActionSubscriber()
   useTcyClaimActionSubscriber()
+  useGenericTransactionSubscriber()
```

Comment:
Think eventually we'll probably want to merge this guy and the below into one and handle all in a master generic transaction hook, but for now trying to keep the diff minimal, just handling RFOX stake/unstake explicitly here for the time being